src/_pytest/_code/source.py

```diff
@@ -152,12 +152,7 @@ def __str__(self):
         return "\n".join(self.lines)
 
     def compile(
-        self,
-        filename=None,
-        mode="exec",
-        flag=generators.compiler_flag,
-        dont_inherit=0,
-        _genframe=None,
+        self, filename=None, mode="exec", flag=0, dont_inherit=0, _genframe=None
```

Comment:
(harder to see the change here since black reformatted it, essentially this:

```diff
-    flag=generators.compiler_flag,
+    flag=0,
```

this is the same as the default builtin `compile(...)` function -- and in versions newer than python2.3 the `generators` flag is ignored so the two are equivalent